tools/cpp/windows_cc_configure.bzl

```diff
@@ -557,80 +610,80 @@ def _get_msvc_vars(repository_ctx, paths):
         write_builtin_include_directory_paths(repository_ctx, "msvc", [], file_suffix = "_msvc")
         msvc_vars = {
             "%{msvc_env_tmp}": "msvc_not_found",
-            "%{msvc_env_path}": "msvc_not_found",
             "%{msvc_env_include}": "msvc_not_found",
```

Comment:
`"%{msvc_env_include_" + target_arch + "}"`?

Reply:
Also `msvc_env_tmp`, `msvc_cxx_builtin_include_directories` should not be override if arm or arm64 toolchains are not found.

Reply:
Will it be better if I added `_{target_arch}` at the end of these variables' names so they won't be overridden by missing toolchains?

Reply:
Yes, I think that makes sense.